server/filters/grid-classes.js

```diff
@@ -0,0 +1,10 @@
+export default function gridClasses(sizes = {}) {
```

Comment:
What's the point of setting the default? People should be required to send a set of sizes no?